slack/1.0/modules/ROOT/pages/slack-connector-reference.adoc

```diff
@@ -0,0 +1,13146 @@
+= Slack Connector 1.0 Reference - Mule 4
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for Slack (Slack Connector) enables access to the Slack platform. This connector exposes all of the operations provided by Slack's REST API.
+
+
+== Configurations
+
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+|Name | String | The name for this configuration. Connectors reference the configuration with this name. | | *x*
+| Connection a| <<Config_SlackAuth, Slack Auth Connection Provider>>
+ | Supported connection types for this configuration. | | *x*
+| Response Timeout a| Number |  Timeout for requests to the remote service. |  1 | 
+| Response Timeout Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  Time unit that qualifies the Response Timeout. |  MINUTES | 
+| Streaming Type a| Enumeration, one of:
+
+** AUTO
+** ALWAYS
+** NEVER |  Defines whether to send the request using streaming. If the value is set to `AUTO` the best strategy is automatically determined based on the content of the request. |  AUTO | 
+| Name a| String |  The identifier of this element used to reference it in other components |  | *x*
+| Expiration Policy a| <<ExpirationPolicy>> |  Configures the minimum amount of time that a dynamic configuration instance can remain idle before the runtime considers it eligible for expiration. This does not mean that the platform will expire the instance at the exact moment that it becomes eligible. The runtime will actually purge the instances when it sees it fit. |  | 
```

Comment:
This is the old Expiration Policy description. We have a proposed new one on our spreadsheet (not that it's much better).